### DISCUSSION TITLE
Expose gluetun to WAN

### DISCUSSION THREAD

**Discussion by seb3point0** [qdm12/gluetun#507] [Q&A] [answered]
Hello 👋 I want to use `gluetun` on a Digital Ocean droplet as a simple HTTP proxy which tunnels trafic through the VPN Client. The issue I'm having is getting the HTTP proxy to be exposed to the Internet (WAN). I couldn't find definitive docs on how to do this.

- Are there any built-in options which enable this?
- If not, what the easiest recommended solution (ex: routing IP tables, NGINX reverse proxy) ?

Ideally, I'm looking for a low-setup and containerized solution so these proxies can be spun up easily.

**Comment by qdm12** [comment, marked as answer]
Hey there, do you mean to http proxy through `<server-ip>:<proxy-port>`? That should work by just publishing the http proxy port with docker. If it doesn't what error or roadblock are you getting?

Note the http proxy connection isn't encrypted between your client and gluetun, so if you're not in the same network, you should use either:

- the shadowsocks proxy built-in with a Shadowsocks client (as it's encrypted)
- VPN to your digital ocean server and then http proxy to gluetun

**Reply by seb3point0** [reply]
Yes, I want to access the proxy through `<wan-server-ip>:<proxy-port>`. This didn't seem to work when I ran the default `docker-config.yml` config. Are there additional options needed to enable this?

**Reply by qdm12** [reply]
By docker-config.yml you mean the docker-compose.yml of the repository right?

Have a look at https://github.com/qdm12/gluetun/wiki/Environment-variables#http-proxy by default everything is off. You can also verify the settings logged by the container at start which will log it if it's enabled or not.

**Reply by seb3point0** [reply]
Yes, I meant `docker-compose.yml` 🤦‍♂️

Ok that worked... I didn't know the proxies were off by default. Thanks!!
